src/mmw/js/src/data_catalog/views.js

```diff
@@ -273,6 +273,7 @@ var TabContentView = Marionette.LayoutView.extend({
 
         // CUAHSI does not support pagination, but others do
         if (this.model.id !== 'cuahsi') {
+            this.resultRegion.$el.addClass('paginated');
```

Comment:
Consider adding this class conditionally in the template.

This also might be a good case for having an `is_pageable` attribute here on the frontend's model like we have on the backend.